macros/no_anomoly.sql

```diff
@@ -0,0 +1,29 @@
+{% test no_anomoly(model) %}
```

Comment:
Should we think about allowing for conditional enabling/disabling this test in a config block with a project var and maybe having it defaulted to disabled?